### PR TITLE
Remove GockTests from integration_test.go

### PR DESCRIPTION
**Pull request by whatsadebugger**

### CODE REVIEW

**Comment by whatsadebugger** [comment]
#245

File: internal/cltest/cltest.go [hunk 1]
```diff
@@ -364,6 +357,17 @@ func CreateJobRunViaWeb(t *testing.T, app *TestApplication, j models.JobSpec, bo
 	return jr
 }
 
+// NewJobHelloWorld creates a JobSpec with the given MockServer Url
+func NewJobHelloWorld(t *testing.T, app *TestApplication, url string) models.JobSpec {
+	j, _ := NewJobWithWebInitiator()
+	t1 := NewTask("httpget", fmt.Sprintf(`{"url":"%v"}`, url))
+	t2 := NewTask("JsonParse", `{"path":["last"]}`)
```
[the remark this answers is not in the document]
Comment (whatsadebugger) [reply internal/cltest/cltest.go]: Yea I think so.

File: internal/cltest/cltest.go [hunk 2]
```diff
@@ -20,7 +20,6 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/gin-gonic/gin"
 	"github.com/gorilla/websocket"
-	"github.com/h2non/gock"
```
[the remark this answers is not in the document]
Comment (whatsadebugger) [reply internal/cltest/cltest.go]: haha :+1:

File: web/integration_test.go [hunk 3]
```diff
@@ -36,21 +35,16 @@ func TestIntegration_Scheduler(t *testing.T) {
 }
 
 func TestIntegration_HelloWorld(t *testing.T) {
-	gock.EnableNetworking()
-	defer cltest.CloseGock(t)
+	tickerResponse := `{"high": "10744.00", "last": "10583.75", "timestamp": "1512156162", "bid": "10555.13", "vwap": "10097.98", "volume": "17861.33960013", "low": "9370.11", "ask": "10583.00", "open": "9927.29"}`
+	mockServer, cleanup := cltest.NewHTTPMockServer(t, 200, "GET", tickerResponse, func(body string) {})
+	defer cleanup()
```
Comment (dimroc) [inline web/integration_test.go]: The second value in the tuple isn't `cleanup`, it's `assertCalled`. Let's change that name and then we're good to go 👍 .

**Pull request merged**